CHANGELOG.md

```diff
@@ -1,9 +1,20 @@
-# dbt_amazon_ads v0.UPDATE.UPDATE
+# dbt_amazon_ads v0.2.0
+## 🚨 Breaking changes
+- This release is labeled breaking since changes made in the [dbt_amazon_ads_source](https://github.com/fivetran/dbt_amazon_ads_source) are breaking. Note that columns in this package are unchanged.
+- Further details are available in the [dbt_amazon_ads_source CHANGELOG](https://github.com/fivetran/dbt_amazon_ads_source/blob/main/CHANGELOG.md).
+## 🎉 Features
+[PR #6](https://github.com/fivetran/dbt_amazon_ads/pull/6) includes the following updates:
```

Comment:
Since there is only one PR for these changes, you can most likely be comfortable putting this section right at the top of the CHANGELOG entry. This way you know all the below changes relate to this PR, and it doesn't need to be added in each individual section.

Reply:
done